iphone/Classes/TiFilesystemFileProxy.m

```diff
@@ -153,6 +153,37 @@ -(id)spaceAvailable:(id)args
 	return [resultDict objectForKey:NSFileSystemFreeSize];
 }
 
+-(id)getProtectionKey:(id)args
+{
+	NSError *error = nil;
+	NSDictionary * resultDict = [fm attributesOfItemAtPath:path error:&error];
+	if (error!=nil) return NUMBOOL(NO);
```

Comment:
If there's an error, maybe we should print it out to the console. Also, document that if this fails, it will return `false` (or `0`)